x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -147,6 +147,9 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
         const [coreStart, startPlugins] = await core.getStartServices();
         const subPlugins = await this.startSubPlugins(this.storage, coreStart, startPlugins);
         const { renderApp } = await this.lazyApplicationDependencies();
+        if (startPlugins.timelines && this._store) {
```

Comment:
we do not need that anymore because of this commit https://github.com/elastic/kibana/pull/110472/commits/da68c5db89babf7da6ef13189728bb62865cffc6